tests/test_apply.py

```diff
@@ -248,8 +250,8 @@ def test_two_qubit_parameters(self, init_state, op, func, theta, tol):
 
         dev._obs_queue = []
 
-        res = np.abs(dev.state) ** 2
-        expected = np.abs(func(theta) @ state) ** 2
+        res = dev.state
+        expected = func(theta) @ state
```

Comment:
A bit weird that the probs were tested instead of the states. :thinking:

Reply:
Yeah was also wondering about that 🤔